dashboard/src/components/resource-list/ResourceListView.js

```diff
@@ -539,6 +549,31 @@ export const UsageTab = ({ usage, children }) => {
   );
 };
 
+const NoDataMessage = ({ type }) => {
+  const classes = useStyles();
+
+  function redirect() {
+    window.location.href = "https://docs.featureform.com/quickstart";
+  }
+  return (
+    <Container>
+      <div className={classes.noDataPage}>
+        <Typography variant="h4">No {type} Registered</Typography>
+        <Typography variant="body1">
+          Looks like your organization hasn't registered one{" "}
```

Comment:
There are no visible Xs in your organization. Check out our docs for step by step instructions to create one.